rest/admin_api_test.go

```diff
@@ -1145,7 +1145,7 @@ func TestDBOfflineSingleResync(t *testing.T) {
 		t.Skip("skipping test in short mode")
 	}
 
-	rt := NewRestTester(t, nil)
+	rt := NewRestTester(t, &RestTesterConfig{noAdminParty: false})
```

Comment:
This test doesn't look like it requires GUEST access - as discussed previously, there are tests where all requests could be routed through the admin port; this appears to be one of those cases. rt.CreateDoc looks like the only usage that's not already doing that, I expect rt.createDoc can just be changed to use the admin port.

Can you please review all tests that you've force-enabled GUEST to ensure it's strictly needed?

Reply:
Yeah, this makes sense, sure let's identify the tests where all requests could be routed through the admin port and review the test where guest access is forced.